### ISSUE TITLE
[@thi.ng/testament] cli wrapper can not find cli.js due to linux using symlink in node_modules/.bin directory

### ISSUE THREAD

**Issue by reedho** [thi-ng/umbrella#349]
Using npm (8.5.5), node (v16.13.0), Fedora Linux 35 (Workstation Edition).

```bash
npx testament test

CustomError: Cannot find module 'PROJECT_DIR/node_modules/cli.js' imported from PROJECT_DIR
...
```

Looking at the wrapper and checked that node_modules/.bin/testament is a symlink, the problem can be fixed by using realpath, e.g.

```js
import { spawn } from "child_process";
import { join } from "path";
import { realpathSync } from "fs";
const [node, toolsDir, ...args] = process.argv;

//const cli = join(toolsDir, "..", "..", "cli.js");
const cli = join(realpathSync(toolsDir), "..", "..", "cli.js");
...
```

**Comment by reedho** [comment]
Thank you Sir. 🙏

**Comment by postspectacular** [comment]
Just released as 0.2.7 -  thank you again!
